src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -27,7 +28,15 @@ public static void printLongestWordTop100() throws IOException {
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        // TODO 이 부분에 구현한다.
+        List<String> longestWords = words.stream()
+                .filter(it -> it.length() > 12)
+                .distinct()
+                .sorted((o1, o2) -> Integer.valueOf(o2.length()).compareTo(Integer.valueOf(o1.length())))
+                .map(String::toLowerCase)
+                .collect(Collectors.toList());
```

Comment:
```suggestion
                .limit(100)
                .map(String::toLowerCase)
                .forEach(System.out::println);
```
컬렉션의 subList 메서드를 사용하지 않고 바로 스트림에서 해결하시는 방법도 있겠네요  😊

Reply:
limit 이 있었군요. stream 에서 바로 해결하는것이 좋겠네요 👍